src/validate.js

```diff
@@ -428,8 +433,21 @@ function F_VALIDATE(disabledHL) {
 		/** @type {Array.<SimpleRESTRICTION>} */
 		this.$restrictions = null;
 
+		/** @type {string} */
+		this.$name = "";
+		/** @type {string} */
+		this.$brand = "";
+		/** @type {string} */
+		this.$objectID = objID;
```

Comment:
The ID of a venue within WME is a dot seperated string containing 3 numbers. Like `"182518064.1825246174.289659"`

Why that is? I don't know, but that are the IDs.

Reply:
Sorry, I did not know that. So, if the venue ID is a string, but the segmentID is a number, we should use pluses (`+`) very carefully, because `+"1.2.3"` is always `NaN`.

The type of objectID should be `string|number` I guess. With this new info, I will thoroughly look through the changes again...

Reply:
Yes those sneaky little pluses before the variable name..... And I was wondering why my strings "suddenly" broke! We both learned haha. I'm looking forward to your comments, I did rework things a bit already.